deploy/10_1_gold_gbp_stablecoins.ts

```diff
@@ -0,0 +1,157 @@
+import { to_d12 } from "../utils/NumbersHelpers";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const initialDeployBDStables = Object.values(bdStablesContractsDetails).filter(stableDetails => ["bXAU", "bGBP"].includes(stableDetails.symbol));
+  for (const stableDetails of initialDeployBDStables) {
+    console.log(`Starting deployment: ${stableDetails.fiat} stable - ${stableDetails.symbol}`);
+
+    const deployer = await getDeployer(hre);
+    const treasury = await getTreasury(hre);
+
+    const bdx = await getBdx(hre);
+    const bdPoolLibraryDeployment = await hre.ethers.getContract("BdPoolLibrary");
+
+    await hre.deployments.deploy(stableDetails.symbol, {
+      from: deployer.address,
+      proxy: {
+        proxyContract: "OptimizedTransparentProxy",
+        execute: {
+          init: {
+            methodName: "initialize",
+            args: [stableDetails.name, stableDetails.symbol, treasury.address, bdx.address, constants.initialBdstableMintingAmount(hre.network.name)]
+          }
+        }
+      },
+      contract: "BDStable",
+      args: []
+    });
+
+    const bdstable = (await hre.ethers.getContract(stableDetails.symbol)) as BDStable;
+    console.log(`${stableDetails.symbol} deployed to:`, bdstable.address);
+
+    await hre.deployments.deploy(stableDetails.pools.weth.name, {
+      from: deployer.address,
+      proxy: {
+        proxyContract: "OptimizedTransparentProxy",
+        execute: {
+          init: {
+            methodName: "initialize",
+            args: [
+              bdstable.address,
+              bdx.address,
+              formatAddress(hre, constants.wETH_address[hre.network.name]),
+              constants.wETH_precision[hre.network.name],
+              true
+            ]
+          }
+        }
+      },
+      contract: "BdStablePool",
+      args: [],
+      libraries: {
+        BdPoolLibrary: bdPoolLibraryDeployment.address
+      }
+    });
+
+    const bdstable_weth_BdStablePool = (await hre.ethers.getContract(stableDetails.pools.weth.name)) as BdStablePool;
+    console.log(`${stableDetails.pools.weth.name} pool deployed to:`, bdstable_weth_BdStablePool.address);
+
+    console.log(`Set ${stableDetails.pools.weth.name} pool parameters`);
+    const [weth_pool_ceiling, weth_redemption_delay, weth_minting_fee, weth_redemption_fee, weth_buyback_fee, weth_recollat_fee] = await Promise.all([
+      bdstable_weth_BdStablePool.pool_ceiling(),
+      bdstable_weth_BdStablePool.redemption_delay(),
+      bdstable_weth_BdStablePool.minting_fee(),
+      bdstable_weth_BdStablePool.redemption_fee(),
+      bdstable_weth_BdStablePool.buyback_fee(),
+      bdstable_weth_BdStablePool.recollat_fee()
+    ]);
+    await (
+      await bdstable_weth_BdStablePool.setPoolParameters(
+        weth_pool_ceiling,
+        to_d12(0.03), // 3% Bonus rate
```

Comment:
Let's move it to a const somewhere

Reply:
this is a specific definition for the deployment script, it doesn't make sense to reuse with other usages since if we will change this constant in the future we still want this value here to remain at 3% since this is how we deployed it.

Reply:
+1